apps/api/src/report/entity/milestone-duration.entity.ts

```diff
@@ -0,0 +1,259 @@
+import { ViewColumn, ViewEntity } from 'typeorm';
+import { HiredApplicantMilestoneEntity } from './hired-applicant-milestone.entity';
+
+// the expression is given by reference
+// because we don't sync schema and the view table should have been created by a migration
```

Comment:
could you expand on this? Is this why we need the full query in both `expression` and the `migration`?